File: 469a_iwbtg.py
from sys import stdin, stdout


def input():
    return stdin.readline().strip()


def print(string):
    return stdout.write(str(string) + "\n")


def main():
    n = int(input())
    x = [int(a) for a in input().split()]
    y = [int(a) for a in input().split()]
    print("I become the guy." if len(
        set(x[1:] + y[1:])) == n else "Oh, my keyboard!")


if __name__ == "__main__":
    main()

File: 469a_iwbtg.js
"use strict";
const readline = require("readline");
let rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
    terminal: false,
});
let lines = [];
rl.on("line", (line) => {
    lines.push(line);
});
rl.on("close", () => {
    let n = parseInt(input());
    let x = input().split(" ").map(Number);
    let y = input().split(" ").map(Number);
    console.log(
        `${
            new Set(x.slice(1).concat(y.slice(1))).size === n
                ? "I become the guy."
                : "Oh, my keyboard!"
        }`
    );
    rl.close();
});
let currentLine = 0;
function input() {
    return lines[currentLine++];
}
